test/plugin/test_out_google_cloud.rb

```diff
@@ -418,11 +418,11 @@ def convert_subfields_to_strings(full_hash, fields_to_convert)
     ])
   end
 
-  # 'responseSize' and 'requestSize' are Integers in the gRPC protos, yet
-  # Strings in REST API client libraries.
+  # 'responseSize', 'requestSize' and 'cacheFillBytes' are Integers in the gRPC
+  # protos, yet Strings in REST API client libraries.
   def http_request_message
     convert_subfields_to_strings(
-      HTTP_REQUEST_MESSAGE, %w(responseSize requestSize))
+      HTTP_REQUEST_MESSAGE, %w(responseSize requestSize cacheFillBytes))
```

Comment:
Do you want to keep these alphabetical as well?

Reply:
Done